rocAL/include/pipeline/context.h

```diff
@@ -27,10 +27,10 @@ THE SOFTWARE.
 #include "pipeline/master_graph.h"
 
 struct Context {
-    explicit Context(size_t batch_size, RocalAffinity affinity, int gpu_id, size_t cpu_thread_count, size_t prefetch_queue_depth, RocalTensorDataType output_tensor_type) : affinity(affinity),
-                                                                                                                                                                            _user_batch_size(batch_size) {
+    explicit Context(size_t batch_size, RocalAffinity affinity, int gpu_id, size_t cpu_thread_count, size_t prefetch_queue_depth, RocalTensorDataType output_tensor_type, RocalBatchPolicy last_batch_policy, bool last_batch_padded) : affinity(affinity),
```

Comment:
in the same enum, we can combine those 2 cases so there will be only one extra parameter for the context

Reply:
Combined both parameters into a pair variable for the constructor - std::pair<RocalBatchPolicy, bool>